### PR TITLE
Revert "Revert "Non default""

### PR DESCRIPTION
**Pull request by double-beep**
Reverts double-beep/batch#5

### CODE REVIEW

**Comment by double-beep** [comment]
Just for completing this pull request.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Copyright 2018
 Batch experiments
+Assigned to sb.
+No contributors needed.
+Batch experiments starting soon.
```
Comment (helper-beep) [inline README.md]: This pull request was needed.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 Copyright 2018
 Batch experiments
```
Comment (helper-beep) [inline README.md]: Standard.

File: README.md (hunk 1 above)
Comment (helper-beep) [inline README.md]: So, this was it!

**Pull request merged**